Package upgrade needed

We get this warning when we compile the latest FSLint:

```
/builds/B2R2/B2R2/FSLint/src/FSLint.LanguageServer/B2R2.FSLint.LanguageServer.fsproj : warning NU1903: Package 'Nerdbank.MessagePack' 1.0.2 has a known high severity vulnerability, https://github.com/advisories/GHSA-2cwq-pwfr-wcw3 [/builds/B2R2/B2R2/FSLint/FSLint.slnx]
```

--- a/src/FSLint.LanguageServer/B2R2.FSLint.LanguageServer.fsproj
+++ b/src/FSLint.LanguageServer/B2R2.FSLint.LanguageServer.fsproj
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>net10.0</TargetFramework>
-    <NoWarn>$(NoWarn);NU1902</NoWarn>
+    <NoWarn>$(NoWarn);NU1902;NU1608</NoWarn>
   </PropertyGroup>
 
   <ItemGroup>
@@ -11,7 +11,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="StreamJsonRpc" Version="2.24.84" />
+    <PackageReference Include="StreamJsonRpc" Version="2.25.25" />
     <PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
   </ItemGroup>
 

--- a/src/FSLint.Tests/B2R2.FSLint.Tests.fsproj
+++ b/src/FSLint.Tests/B2R2.FSLint.Tests.fsproj
@@ -5,7 +5,7 @@
     <LangVersion>latest</LangVersion>
     <ImplicitUsings>enable</ImplicitUsings>
     <Nullable>enable</Nullable>
-    <NoWarn>NU1605</NoWarn>
+    <NoWarn>NU1605;NU1608</NoWarn>
   </PropertyGroup>
 
   <ItemGroup>

--- a/src/FSLint/B2R2.FSLint.fsproj
+++ b/src/FSLint/B2R2.FSLint.fsproj
@@ -16,7 +16,7 @@
     <RepositoryType>git</RepositoryType>
     <PackageProjectUrl>https://b2r2.org/B2R2</PackageProjectUrl>
     <PackageTags>Binary;B2R2;lint</PackageTags>
-    <NoWarn>NU1605</NoWarn>
+    <NoWarn>NU1605;NU1608</NoWarn>
   </PropertyGroup>
 
   <PropertyGroup>

